remarkable.tablet

```diff
@@ -0,0 +1,29 @@
+# reMarkable 
+#
+# Button Map: TODO
+#
+# Default button assignments: TODO
+#
+# stylus with two buttons and no eraser
+# Pen active area: 8.25 x 6.125 in
+
+[Device]
+Name=reMarkable tablet 1
+DeviceMatch=usb:056a:0004
+Class=Bamboo
+Width=6
+Height=8
+IntegratedIn=
+#Layout=
+Styli= #FIXME
```

Comment:
`Styli` should be set to `0xffffe;0xffffff;`.

See [here](https://github.com/linuxwacom/libwacom/blob/master/data/wacom.example#L105).